test/fixtures/comparison-files/lib/viewers/ResultsViewer.test.ts/four-identical-violations-details.txt

```diff
@@ -0,0 +1,33 @@
+*DRAFT*: Found 4 violation(s) across 1 file(s):
+[2m===[22m [1m*DRAFT*: 1. stub1RuleA[22m
```

Comment:
Do we need to lock in the ansi styling in our automated tests here - or is this something we should just manually look at once and determine it is fine? I think users would let us know if it was off anyway.

So I say keep the styling for the StylingUtil tests... but here, for the sake of readability consider stripping off the ansi styling before saving your gold file. Then when testing, you can just strip off the styling from the actual value as well before doing the comparison.

This way we have some nice files to point to an example text for what our output should look like.

Reply:
Reasonable. I'll make it happen.